test/spoom/cli/deadcode_test.rb

```diff
@@ -6,6 +6,10 @@
 module Spoom
   module Cli
     class DeadcodeTest < TestWithProject
+      def setup
+        @project.bundle_install!
+      end
```

Comment:
Flakey errors seems to go away with this. We don't actually run it for each project, it's too slow. We can keep fixing them like this as they come up.